src/howard/howard/factories/issuers/invoice.py

```diff
@@ -0,0 +1,99 @@
+"""Factories for invoice issuers."""
+
+from django.utils import timezone
+
+import factory
+import factory.fuzzy
+from howard.issuers import invoice
+
+
+class InvoiceMetadataFactory(factory.Factory):
+    """Factory for the Metadata class."""
+
+    class Meta:
+        """Meta class for the Metadata class."""
+
+        model = invoice.Metadata
+
+    id = factory.Sequence(
+        lambda n: (f"{timezone.now().strftime('%Y%m%d%H%M%S')}-" f"{n:08d}")
```

Comment:
Not sure why there are two f strings?
```suggestion
        lambda n: (f"{timezone.now().strftime('%Y%m%d%H%M%S')}-{n:08d}")
```

Reply:
Formatting on save ...

Reply:
#UseVIM